src/inmanta/db/versions/v202302270.py

```diff
@@ -0,0 +1,34 @@
+"""
+    Copyright 2022 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+
+from asyncpg import Connection
+
+
+async def update(connection: Connection) -> None:
+    schema = """
+    CREATE TYPE auth_method AS ENUM('database', 'oidc');
+
+    CREATE TABLE IF NOT EXISTS public.user(
+        id uuid PRIMARY KEY,
+        username VARCHAR NOT NULL UNIQUE ,
+        password_hash VARCHAR NOT NULL,
```

Comment:
Is there a specific reason why the default collation that PostgreSQL uses wouldn't be sufficient here?

Reply:
I honestly don't know. I do seem to recall the "default" was a very loose concept here. I think PostgreSQL itself doesn't have one and it falls back to something ill defined, Markus wasn't positive about it but I don't recall the details. My takeaway was that generally you should always use an explicit collation, but I could just as well have misunderstood that. You didn't understand that the same way?

Reply:
I mainly understood that you have to use collations if you want to have the output sorted in a specific way. I am not sure what the default collation does, but I don't really see why we would want to have the usernames sorted in a specific way. To me it's important that the API returns the usernames in a consistent order. Whether capital letters come before small letter or the other way around, etc. is not that important to me.

Reply:
I agree with that sentiment, so since I can't come up with a concrete reason to use the collation, let's leave it out for now. If it turns out to have been important it will come back up.